telegram/ext/conversationhandler.py

```diff
@@ -358,10 +366,13 @@ def update_state(self, new_state, key):
 
     def _trigger_timeout(self, bot, job):
         self.logger.debug('conversation timeout was triggered!')
-        del self.timeout_jobs[job.context['current_conversation']]
+        conversation_key = job.context.conversation_key
+        update = job.context.update
+        dispatcher = job.context.dispatcher
```

Comment:
nitpicking again: why the over verbosity of the above 3 lines? why not just call `job.context.<something>` when needed?